site/layouts/core.hbs

```diff
@@ -11,7 +11,7 @@
 		{{>metadata}}
 		{{>headresources}}
 	</head>
-	<body {{#if pageclass}}class="{{pageclass}}" {{/if}}vocab="http://schema.org/" typeof="WebPage">
+	<body {{#if pageclass}}class="{{pageclass}}" {{/if}}vocab="https://schema.org/" typeof="WebPage">
```

Comment:
The schema.org vocabulary should use "http://" scheme, not the "https://" 

reference: https://www.w3.org/2011/rdfa-context/rdfa-1.1

Reply:
I changed the vocab attribute's protocol was due to the [question/answer about HTTPS on schema.org's FAQ](https://schema.org/docs/faq.html#19).

**Excerpt:**
> **Should we write ``https://schema.org`` or ``http://schema.org`` in our markup?**
> 
> There is a general trend towards using ``https`` more widely, and you can already write ``https://schema.org`` in your structured data. Over time we will migrate the schema.org site itself towards using https: as the default version of the site and our preferred form in examples. However ``http://schema.org`` -based URLs in structured data markup will remain widely understood for the forseeable future and there should be no urgency about migrating existing data. This is a lengthy way of saying that both ``https://schema.org`` and ``http://schema.org`` are fine.

Thoughts?

Reply:
Reverted this change as per our follow-up discussion.